pkg/server/interfaces/api/application.go

```diff
@@ -521,7 +521,7 @@ func (c *application) GetWebServiceRoute() *restful.WebService {
 		Param(ws.PathParameter("appName", "identifier of the application.").DataType("string").Required(true)).
 		Param(ws.PathParameter("workflowName", "identifier of the workflow").DataType("string")).
 		Returns(200, "OK", apis.EmptyResponse{}).
-		Writes(apis.EmptyResponse{}).Do(returns200, returns500))
```

Comment:
For interface returning EmptyResponse. We can keep the `returns200` and remove the 
```go
Returns(200, "OK", apis.EmptyResponse{}).
```